packages/teams-js/webpack.config.js

```diff
@@ -52,5 +64,34 @@ module.exports = {
       removeSource: true,
       outputAsModuleFolder: true,
     }),
+
+    // https://www.npmjs.com/package/webpack-subresource-integrity
+    new SubresourceIntegrityPlugin({ enabled: true }),
+
+    // Webpackmanifest produces the json file containing asset(JS file) and its corresponding hash values(Example: https://github.com/waysact/webpack-subresource-integrity/blob/main/examples/webpack-assets-manifest/webpack.config.js)
+    new WebpackAssetsManifest({
+      integrity: true,
+      integrityHashes: ['sha384'],
```

Comment:
With this change are you still generating all the different hashes (e.g., sha256, sha384, sha512, etc.)? Or is it only sha384 now?

Reply:
No it will generate only sha384

Reply:
Okay thanks